src/app/clusters/mode-select-server/static-supported-modes-manager.h

```diff
@@ -24,6 +24,10 @@
 #include <map>
 #include <vector>
 
+#ifndef ENDPOINTS_WITH_MODES
+#define ENDPOINTS_WITH_MODES 1
```

Comment:
Why is this not using the existing define, for static endpoints?  And for dynamic ones this would not work.